Applications/Games/Overwatch/Online/script.js

```diff
@@ -1,28 +1,34 @@
 include(["engines", "wine", "quick_script", "online_installer_script"]);
-include(["engines", "wine", "plugins", "csmt"]);
+// include(["engines", "wine", "plugins", "csmt"]);
 include(["engines", "wine", "plugins", "windows_version"]);
+include(["engines", "wine", "plugins", "override_dll"]);
 include(["engines", "wine", "verbs", "vcrun2015"]);
 include(["engines", "wine", "verbs", "corefonts"]);
+include(["engines", "wine", "verbs", "dxvk"]);
+
+BUG: https://github.com/PhoenicisOrg/scripts/issues/792
 
 var installerImplementation = {
     run: function () {
         new OnlineInstallerScript()
             .name("Overwatch")
             .editor("Blizzard")
             .applicationHomepage("http://www.playoverwatch.com/")
-            .author("ImperatorS79")
+            .author("ImperatorS79, kreyren")
             .url("https://eu.battle.net/download/getInstaller?os=win&installer=Overwatch-Setup.exe")
         //The checksum is different each time you download
             .wineVersion(LATEST_STAGING_VERSION)
             .wineDistribution("staging")
-            .wineArchitecture("amd64")
+            .wineArchitecture("amd64") // Multilib preffered? (without setting wine architecture)
             .category("Games")
-            .executable("Overwatch.exe")
+            .executable("Battle.net.exe") // affect ed by bug 
             .preInstall(function (wine/*, wizard*/) {
-                wine.windowsVersion("winxp");
-                wine.vcrun2015();
-                wine.corefonts();
-                wine.enableCSMT();
+                wine.windowsVersion("win7");
+                wine.vcrun2015(); // untested, should be mandatory
+                wine.corefonts(); // Untested, for fonts?
+                // wine.enableCSMT(); // Commented out since it's not tested
```

Comment:
Please remove all the commented line (despite the checksum one)

Reply:
@ImperatorS79 Insane, needs more testing.

Reply:
Believe me. CSMT for example is automatically enabled in wine nowadays. And it is only related to wine3d, so if you use DXVK -> no need for csmt "verb".

Reply:
@ImperatorS79 Contribute it then yourself i don't want to be responsible for that commit since i don't have required informations to confirm it's sanity.. 

Note that I'm unable to run OverWatch on platinum at the moment only Pre-Platinum with freezes every few minutes which is not acceptable. Working on it assuming limited time based on set priority task list.